client/src/components/Dashboard/Dashboard.jsx

```diff
@@ -111,7 +111,7 @@ function DashboardJobs() {
       }
       contentShouldScroll
     >
-      <JobsView />
+      <JobsView includeSearchbar={false} />
```

Comment:
I'm not certain what is causing it but the dashboard i have no jobs:

![Screenshot 2023-02-15 at 3 00 36 PM](https://user-images.githubusercontent.com/8287580/219159067-920c3867-c2d6-4462-86c4-62f9edfc14f1.png)

Reply:
Can you check to see if you have the latest version of the branch pulled down? I pushed a fix for that this morning

Reply:
yes! that did the trick. thanks!